Fronter/Source/Frames/Tabs/PathsTab.cpp

```diff
@@ -95,11 +95,13 @@ void PathsTab::initializePaths()
 			auto buf = Utils::GetCurrentDirectoryWString();
 #if defined __WIN32__
 			auto currentDirectory = buf + '\\';
+			filePath = currentDirectory + Utils::convertUTF8ToUTF16(file.second->getSearchPath() + '\\' + file.second->getFilename());
+			initialPath = currentDirectory + Utils::convertUTF8ToUTF16(file.second->getSearchPath()) + '\\');
```

Comment:
There's one extra bracket here. 
`initialPath = currentDirectory + Utils::convertUTF8ToUTF16(file.second->getSearchPath()) + '\\');`
should be
`initialPath = currentDirectory + Utils::convertUTF8ToUTF16(file.second->getSearchPath() + '\\');`